pkg/yurtmanager/webhook/deploymentrender/v1alpha1/deploymentrender_handler.go

```diff
@@ -24,13 +24,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
+	yurtClient "github.com/openyurtio/openyurt/cmd/yurt-manager/app/client"
+	"github.com/openyurtio/openyurt/cmd/yurt-manager/names"
 	"github.com/openyurtio/openyurt/pkg/yurtmanager/webhook/util"
 )
 
 // SetupWebhookWithManager sets up Cluster webhooks. 	mutate path, validatepath, error
 func (webhook *DeploymentRenderHandler) SetupWebhookWithManager(mgr ctrl.Manager) (string, string, error) {
 	// init
-	webhook.Client = mgr.GetClient()
+	webhook.Client = yurtClient.GetClientByControllerNameOrDie(mgr, names.YurtAppOverriderController)
```

Comment:
YurtAppOverrider is deprecated now, so maybe it is a good idea to use `YurtAppSet` client here.

Reply:
Deploymentrender webhook comes with yurtappoverrider. Maybe we can also deprecate this webhook rather than use YurtAppSet client?

Reply:
ok, i got it.